src/test/java/org/takes/rs/RsVelocityTest.java

```diff
@@ -54,4 +56,23 @@ public void buildsTextResponse() throws IOException {
         );
     }
 
+    /**
+     * RsVelocity should close template's InputStream after serving response.
+     * @throws IOException If some problem inside
+     */
+    @Test
+    public void closesTemplateInputStream() throws IOException {
+        final String template = "hello, world!";
+        final MockInputStream templateStream = new MockInputStream(template);
+        MatcherAssert.assertThat(
+            IOUtils.toString(
+                new RsVelocity(
+                    templateStream,
+                    Collections.<CharSequence, Object>emptyMap()
+                ).body()
+            ),
+            Matchers.equalTo(template)
+        );
+        MatcherAssert.assertThat(templateStream.isClosed(), Matchers.is(true));
```

Comment:
@Happy-Neko Better to mock `InputStream` with Mockito here, since it's the only behavior we're testing.